server.js

```diff
@@ -23,11 +23,23 @@ const comments = require('./routes/comments');
 const shippingMethods = require('./routes/shippingMethods');
 const paymentMethods = require('./routes/paymentMethods');
 const partners = require('./routes/partners');
+const { nextTick } = require('process');
 // const mainRoute = require("./routes/index");
 
 const app = express();
 
 // Body parser middleware
+
+app.use((reg, res, next) => {
+  res.setHeader('Access-Control-Allow-Origin', '*');
+  res.setHeader(
+    'Access-Control-Allow-Methods',
+
+    'OPTIONS, GET, POST, PUT, PATCH, DELETE' // what matters here is that OPTIONs is present
+  );
+  res.setHeader('Access-Control-Allow-Headers', 'Content-Type, Authorization');
+  next();
+});
```

Comment:
this change in backend was not discussed with the team,  please write a brief summary for the whole team in Telegram and tag our Mentor in order to approve this edit to the server file.

Reply:
I am responsible for this change and take responsibility
it just removes the problem with CORS policy